### PR TITLE
Fix selected project test flake

### LINKED ISSUE #2005: Race: ProjectsService.list() sometimes doesn't contain new project
If I visit the `create-project` page directly without going to the projects list first, it's possible that the `ProjectService.list` call does not return with the new project, and the wrong project is selected in the dropdown. This only happens when you visit the `create-project` page first with the new experience turned out, but it also is causing test failures in our merge queue.

screen shot 2017-09-04 at 2 48 33 pm

https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin_web_console/115/console

```
1)  authenticated e2e-user new project when creating a new project should successfully create a new project
  - Error: Element not found: .project-bar option[selected]
Wait timed out after 5149ms
```

### PR DESCRIPTION
**Pull request by spadgett**
Closes #2005

### CODE REVIEW

**Comment by spadgett** [comment]
[test]

**Comment by spadgett** [comment]
[test]

**Comment by spadgett** [comment]
[test]

**Comment by spadgett** [comment]
[test]

**Comment by spadgett** [comment]
[test]

**Comment by spadgett** [comment]
@jeff-phillips-18 I think I've fixed the flakes, PTAL

File: dist/scripts/vendor.js [hunk 1]
```diff
@@ -45157,34 +45152,7 @@ sSortJUIWrapper: "",
 sSortIcon: "",
 sJUIHeader: "",
 sJUIFooter: ""
-}), function() {
```
Comment (benjaminapetersen) [inline dist/scripts/vendor.js]: Does the dist here seem odd?

File: app/scripts/directives/nav.js [hunk 2]
```diff
@@ -329,8 +329,23 @@ angular.module('openshiftConsole')
             });
```
Comment (benjaminapetersen) [inline app/scripts/directives/nav.js]: Pulled & ran it to see if I could replicate the bug.  Haven't quite figured out how this change fixes the test, but +1 😄

**Comment by spadgett** [comment]
Well I've fixed one of two flakes anyway. Investigating the other

File: dist/scripts/vendor.js (hunk 1 above)
Comment (spadgett) [reply dist/scripts/vendor.js]: > Does the dist here seem odd?

The datatables dependency changed.

**Comment by spadgett** [comment]
[test]

**Comment by spadgett** [comment]
I think there is another periodic flake, but this fixes one problem.

[merge]

**Comment by benjaminapetersen** [comment]
Think it flaked again:

```bash 
Error: Playbook execution failed with code 2
++ export status=FAILURE
++ status=FAILURE
+ set +o xtrace
########## FINISHED STAGE: FAILURE: PROVISION CLOUD RESOURCES [00h 10m 20s] ##########
Build step 'Execute shell' marked build as failure
[PostBuildScript] - Execution post build scripts.
```

[merge][severity: blocker]

**Comment by spadgett** [comment]
[merge]

**Comment by spadgett** [comment]
[merge]

**Pull request merged**